plugins/nanospell/plugin.js

```diff
@@ -87,6 +90,16 @@
 				(path.blockLimit ? path.blockLimit.equals(startNode) : true) && // check we don't enter another block-like element
 				(path.block ? path.block.equals(startNode) : true); // check we don't enter nested blocks (special list case since it's not considered a limit)
 
+			// If it's not a rootBlock text node, check to see if we hit a nested block element
+			if (!condition) {
+				if (isNotBookmark(node) &&
+				 	((path.blockLimit && !path.blockLimit.equals(startNode)) ||
```

Comment:
I think we should probably extract `path.blockLimit.equals(startNode)` and  `path.block.equals(startNode)` to variables and use them here.  

Though I'm currently failing to find a concise way to describe it.

startNodeIsBlockLimit & startNodeIsBlock?